es/learn/regex.md

```diff
@@ -1,33 +1,33 @@
 
-In this article, we'll look at the basics of regex, including common use cases, key functions in the `re` module, and some tips to make your expressions both effective and efficient. Whether you're new to programming or looking to refine your pattern-matching skills, regex in Python is a versatile tool well worth learning.  
+En este artículo, veremos los conceptos básicos de regex, incluyendo casos de uso comunes, funciones clave en el módulo `re`, y algunos consejos para hacer tus expresiones tanto efectivas como eficientes. Ya seas nuevo en la programación o busques refinar tus habilidades de coincidencia de patrones, regex en Python es una herramienta versátil que vale la pena aprender.
 
-## Advantages of Regular Expressions in Python
+## Ventajas de las Expresiones Regulares en Python
 
-- **Efficiency in Text Processing:** Quickly search, match, or replace patterns in text.
-- **Versatility:** From data validation to web scraping, regex finds its application.
-- **Simplified Syntax:** Python's [re](https://docs.python.org/3/library/re.html) module simplifies the implementation of regex.
-- **Pattern Matching Power:** Enables matching complex patterns that would be challenging to find with traditional methods.
-  
-## Usage of Regular Expressions in Python
+- **Eficiencia en el Procesamiento de Texto:** Busca, coincide o reemplaza rápidamente patrones en el texto.
+- **Versatilidad:** Desde la validación de datos hasta el web scraping, regex encuentra su aplicación.
+- **Sintaxis Simplificada:** El módulo [re](https://docs.python.org/3/library/re.html) de Python simplifica la implementación de regex.
+- **Poder de Coincidencia de Patrones:** Permite coincidir patrones complejos que serían desafiantes de encontrar con métodos tradicionales.
 
-Regular expressions can be employed in a myriad of scenarios ranging from simple string matching to complex text parsing tasks such as:
+## Uso de Expresiones Regulares en Python
 
-- Data validation (emails, phone numbers)
+Las expresiones regulares pueden emplearse en una miríada de escenarios que van desde la coincidencia de cadenas simples hasta tareas complejas de análisis de texto como:
+
+- Validación de datos (correos electrónicos, números de teléfono)
 - Web scraping
-- Log file analysis
-- Text data pre-processing in Machine Learning tasks
+- Análisis de archivos de registro
+- Pre-procesamiento de datos de texto en tareas de Aprendizaje Automático
 
-For those looking to experiment with Python regex online, numerous platforms and regex testers are available that allow immediate feedback and learning.
+Para aquellos que buscan experimentar con regex de Python en línea, numerosas plataformas y probadores de regex están disponibles que permiten retroalimentación e aprendizaje inmediatos.
 
-### Example: Email Address Found
+### Ejemplo: Dirección de Correo Electrónico Encontrada
 
-```python3
```

Comment:
Don't change `python3` to `python` here. Do you use the most recent updates in the `master` branch as a source of English text?